task_processing/plugins/kubernetes/task_config.py

```diff
@@ -118,6 +121,27 @@ def _valid_volumes(volumes: Sequence["DockerVolume"]) -> Tuple[bool, Optional[st
     return (True, None)
 
 
+def _valid_empty_volumes(volumes: Sequence[EmptyVolume]) -> Tuple[bool, Optional[str]]:
+    for volume in volumes:
+        if not set(volume.keys()) <= VALID_EMPTY_VOLUME_KEYS:
+            return (
+                False,
+                f'Invalid empty volume format, must only contain following keys: '
+                f'{VALID_EMPTY_VOLUME_KEYS}, got: {volume.keys()}'
+            )
+        if 'container_path' not in set(volume.keys()):
```

Comment:
oh, not all the keys are required? i think i'd prefer using `issubset` over the operator version if so - i think this is rare enough that most people (myself included) will need to look at the docs to remember what <= does in a set context